llpc/context/llpcCompiler.cpp

```diff
@@ -177,6 +177,7 @@ enum ShaderReplaceMode {
 };
 
 static ManagedStatic<sys::Mutex> SCompilerMutex;
```

Comment:
Help to do this change: SCompilerMutex -> CompilerMutex, SOptionHash -> OptionHash
I think this is caused by renaming of coding style change

Reply:
Thanks for the suggestion. I had some lines deleted that I hand't intended to remove - fixed now